src/lib/schematics/tsconfig.json

```diff
@@ -16,7 +16,10 @@
     ],
     "baseUrl": ".",
     "paths": {
-      "@angular/cdk/schematics": ["../../../dist/packages/cdk/schematics/"]
+      "@angular/cdk/schematics": [
+        "../../../dist/packages/cdk/schematics/",
```

Comment:
@DevVersion I don't know whether this should still be here. I've left it in more now just in case. Also we have to look into why the CI doesn't fail for these. I took a quick look and it seemed like the TS compilation promise was rejecting as expected.

Reply:
Yeah this is odd. I assumed there are no "non-breaking" failures because the CI passed and `gulp material:build-release:clean` as well. I probably just didn't scroll up the whole log because it exited without errors.

Also, yeah, let's remove `../../../dist/packages/cdk/schematics` here and just refer to the source. I do not recall why we always refer to `dist/packages` (in other entry-points of Material)